### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by pdxjohnny** [01org/ciao#1186]
ciao needs a cool logo! How else am I supposed to showcase it in my powerpoints!?!?!?!
Clear Linux has the nifty penguin ciao needs some logo love too

**Comment by theodorosploumis** [comment]
Hi, I could help with this if you like. I am not a logo designer but I like creating logos and graphics for open source projects. Should I proceed?

Thanks in advanced!

**Comment by theodorosploumis** [comment]
Something like this maybe! Combining the Cloud, letter C and a tree of dots (aka nodes).

![proposals_1-4](https://cloud.githubusercontent.com/assets/1315321/24548830/46b3015a-161f-11e7-9164-16cd6f109e50.png)

**Comment by albertomurillo** [comment]
@theodorosploumis I'm afraid 1,3 & 4 looks like skype

**Comment by theodorosploumis** [comment]
@albertomurillo Not really. Skype uses the letters with a thick border not a cloud as also as a different blue color. On the other hand, if you search for "[blue cloud logo](https://www.google.gr/search?q=cloud+blue+logo&tbm=isch)" on Google there will be lot of "similar" logos... I have created these logos without looking at any other logos just to avoid the above issue.

**Comment by theodorosploumis** [comment]
How about this one then? I would advice to make this logo as simple as possible so it can be displayed even in small sizes.

![g4850](https://cloud.githubusercontent.com/assets/1315321/26686363/5a652e10-46f5-11e7-96d4-10c4142990ac.png)

**Comment by rcaballeromx** [comment]
Have you requested a logo from marketing/branding? Logos must be registered/trademarked and that team has an entire process which must be followed.

**Comment by theodorosploumis** [comment]
No, I did not know about that. And I am not an employee of Intel.
